.github/workflows/ci.yml

```diff
@@ -1323,15 +1323,21 @@ jobs:
     uses: ./.github/workflows/snapshot-docker-version-tag.yml
     secrets: inherit
 
+  # Dynamically extract the stable version from the branch name
+  extract-stable-version:
```

Comment:
For UX reasons I'd love to avoid introducing a 2nd job besides `snapshot-docker-version-tag` to keep the list of jobs and dependencies to a minimum. Could they be merged, given that the commands they execute are not too complicated?